build/bin/vars

```diff
@@ -2,7 +2,8 @@ bin_dir=$(cd $(dirname $0); pwd -P)
 root_dir=$(cd $(dirname $0)/../..; pwd -P)
 tmp_dir=${root_dir}/tmp
 branch=$(git rev-parse --abbrev-ref HEAD)
-real_pmm_version=$(cat ${root_dir}/VERSION)
+real_pmm_version=$(cat ${root_dir}/VERSION | awk -F'-' '{print $1}')
+real_pmm_release=$(cat ${root_dir}/VERSION | awk -F'-' '{print $2}')
```

Comment:
A comment should be present explaining the difference between version and release, and why we awk.